spec/models/provider_spec.rb

```diff
@@ -169,6 +169,16 @@
     it { should_not include(provider2) }
   end
 
+  describe ".with_full_name" do
```

Comment:
```suggestion
  describe "#with_full_name" do
```

Reply:
As with `with_provider_name`, `with_full_name` is a class method so `.` is appropriate.